tools/osx-pkg/scripts/npm_preinstall.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# TODO Can this be done inside the .pmdoc?
+# TODO Can we extract $PREFIX from the installer?
+
+set -e
+
+rm -rf /usr/local/lib/node_modules/npm
```

Comment:
I'll test it then

Can't really use CI machines to test installers (yet) unfortunately, not easily automatable